app/com/gu/floodgate/DynamoDBTable.scala

```diff
@@ -62,6 +62,32 @@ trait DynamoDBTable[T] extends StrictLogging {
     promise.future
   }
 
+  def getItems(id: String, keyName: String) = {
+    val promise = Promise[List[T]]()
+
+    val request = new QueryRequest().withTableName(tableName)
+      .withKeyConditionExpression(s"$keyName = :$keyName")
+      .withExpressionAttributeValues(
+        Map(s":$keyName" -> new AttributeValue().withS(id)))
```

Comment:
Do you need to specify `withScanIndexForward(false)` to get the newest results first? I can't remember which way round the sort order works by default.